src/.github/workflows/testing.yml

```diff
@@ -0,0 +1,25 @@
+name: Test
```

Comment:
Why are there two copies of this file, only the top level one will be used by GitHub.